docs/zh/sql-reference/sql-statements/loading_unloading/ETL/DROP_TASK.md

```diff
@@ -59,3 +60,19 @@ MySQL > SUBMIT /*+set_var(query_timeout=100000)*/ TASK `ctas` AS
 MySQL > DROP TASK `ctas`;
 Query OK, 0 rows affected (0.35 sec)
 ```
+
+删除不存在的任务
+
+- 当未声明 `IF EXISTS` 参数时，删除一个不存在的 task `test_task` 会报错。
+
+```Plain
+MySQL > DROP TASK test_task;
+Query 1 ERROR: Getting analyzing error. Detail message: Task test_task is not exist.
+```
+
+- 当声明 `IF EXISTS` 参数时，删除一个不存在的 `test_task` 不会报错。
+
+```Plain
+MySQL > DROP TASK IF EXISTS test_task;
```

Comment:
According to this example, the syntax should be `DROP TASK [IF EXISTS] `<task_name>` [FORCE]